Units/parser-javascript.r/github-issue-780.d/expected.tags

```diff
@@ -1,3 +1,3 @@
-AnonymousClass6b70cce50101	input.js	/^const Bar = Foo.create({$/;"	c
 Bar	input.js	/^const Bar = Foo.create({$/;"	C
-jump	input.js	/^  jump() {$/;"	m	class:AnonymousClass6b70cce50101
+anonymousObject6b70cce50105	input.js	/^const Bar = Foo.create({$/;"	v
+jump	input.js	/^  jump() {$/;"	m	class:anonymousObject6b70cce50105
```

Comment:
Though `anonymousObject6b70cce50105` is tagged as a variable, it is referred as a class in the tag of `jump`.